### ISSUE TITLE
Support Python 3.7

### ISSUE THREAD

**Issue by abatilo** [abatilo/sanic-swagger#15]
### Subject of the issue
Types in Python 3.7 fail to pass the tests that we currently have. This somewhat relates to #12 

### Steps to reproduce
Switch the python version that's defined in the Pipfile, to use Python 3.7, and then try to run our tests

### Expected behaviour
All of the tests should pass

### Actual behaviour
5 tests fail under Python 3.7

**Comment by hairychris** [comment]
This is the PEP that gets rid of GenericMeta: https://www.python.org/dev/peps/pep-0560/

**Comment by hairychris** [comment]
These "fixes" pass all the tests but I really don't think it's correct. Would you mind taking a look when you get the chance? https://github.com/hairychris/sanic-swagger/tree/patch-2

**Comment by abatilo** [comment]
I would agree that it doesn't feel like the correct approach, but in full transparency, I'm not really sure :/

I'm not sure that I understand what happened with GenericMeta when we moved to Python 3.7

**Comment by c-goosen** [comment]
I am also looking at this. We can add a dependency to keep GenericMeta? https://pypi.org/project/typing/

We can just create a class for GenericMeta?

First of all the fix @hairychris you could import GenericMeta as generic.

`from typing import Generic as GenericMeta`
